osu.Game/Users/UserPanel.cs

```diff
@@ -0,0 +1,205 @@
+                                        },
+                                        new CircularContainer
+                                        {
+                                            Width = 20f,
+                                            RelativeSizeAxes = Axes.Y,
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+                statusBar = new Container
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    RelativeSizeAxes = Axes.X,
+                    Alpha = 0f,
+                    Children = new Drawable[]
+                    {
+                        statusBg = new Box
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Alpha = 0.5f,
+                        },
+                        new FillFlowContainer
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            AutoSizeAxes = Axes.Both,
+                            Spacing = new Vector2(5f, 0f),
+                            Children = new[]
+                            {
+                                new TextAwesome
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Icon = FontAwesome.fa_circle_o,
+                                    Shadow = true,
+                                    TextSize = 14,
+                                },
+                                statusMessage = new OsuSpriteText
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Font = @"Exo2.0-Semibold",
+                                },
+                            },
+                        },
+                    },
+                },
+            };
+
+            cover.Add(new AsyncLoadWrapper(new CoverBackgroundSprite(user)
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                FillMode = FillMode.Fill,
+                OnLoadComplete = d => d.FadeInFromZero(200),
+            }) { RelativeSizeAxes = Axes.Both });
+
+            Status.ValueChanged += displayStatus;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            this.colours = colours;
+        }
+
+        private void displayStatus(UserStatus status)
+        {
+            statusBar.ResizeHeightTo(status == null ? 0f : status_height, 500, EasingTypes.OutQuint);

```

Comment:
`const float transition_duration = 500;`

also i'd consider splitting this out into

```
if (status == null)
{
    ...
}
else
{
    ...
}

to avoid all the `status==null` checks